Sources/Native/DictionaryExtensions.swift

```diff
@@ -3,8 +3,8 @@ import Foundation
 extension Dictionary {
   public func merge(dictionary: [Key: Value]) -> [Key: Value] {
     var newDictionary: [Key: Value] = self
-    for (key, value) in dictionary {
-      newDictionary[key] = value
+    for (dictKey, value) in dictionary {
```

Comment:
Nitpicking here... The `newDictionary` already defines it as `[Key: Value]`, following the convention would make sense.

Reply:
Are you talking about the type `[Key: Value]` or the naming `(dictKey, value)`?